firebase-rules-generator/src/cli.ts

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 
-import * as meow from 'meow';
+import meow from 'meow';
 import * as path from 'path';
-import {buildFile} from './build';
+import {buildFile} from './build.js';
```

Comment:
is the `.js` needed?

Reply:
yes, see this comment: https://github.com/simpleclub/firebase-rules-helper/pull/22#discussion_r1271874942